### ISSUE TITLE
Enable aria-roles Axe rule on Caseflow

### ISSUE THREAD

**Issue by alisan16** [department-of-veterans-affairs/caseflow#12871]
## Description
This task is for enabling the `aria-roles` rule: Ensures all elements with a role attribute use a valid value

## Background/context/resources
See Epic #12884

**Comment by yoomlam** [comment]
Some more background:
- https://a11yproject.com/posts/getting-started-aria/
- https://www.w3.org/TR/wai-aria/#roles_categorization

**Comment by kevmo** [comment]
🔍 Find ~760 elements missing aria roles & add aria roles! However, this _could_ potentially just be a regex find and replace. The epic includes a listing of the number of errors. See the w3 link above for types of aria roles.  --> There will be some investigation.

**Comment by kevmo** [comment]
# Estimation: 5

_[What is this graph?](https://github.com/department-of-veterans-affairs/caseflow/issues/11533)_
```bash
1 | 
2 | |||
3 | ||||||
5 | ||||||
8 | 
```

Potentially tedious ticket with much double-checking.
